.circleci/config.yml

```diff
@@ -73,18 +73,6 @@ jobs:
       - run:
           name: Build Storybook
           command: npm run storybook:ci
-      - run:
```

Comment:
You'll need to store `docs/` to the workspace so that the `release` job can pick it up

Reply:
Complete.